README.md

```diff
@@ -51,6 +51,8 @@ npm i puppeteer-core
 
 `puppeteer-core` is intended to be a lightweight version of puppeteer for launching an existing browser installation or for connecting to a remote one.
 
+See [puppeteer vs puppeteer-core](https://github.com/GoogleChrome/puppeteer/blob/v1.7.0/docs/api.md#puppeteer-vs-puppeteer-core).
```

Comment:
@aslushnikov It seems this link does not work.

Reply:
indeed; addressed in https://github.com/GoogleChrome/puppeteer/pull/3197.